docs/wiki/Home.md

```diff
@@ -20,6 +20,8 @@ If you're unfamiliar with Infrastructure as Code, or wonder how you can use the
 - [Testing](./Testing)
   - [Design](./TestingDesign)
   - [Usage](./TestingUsage)
+- Tools/Utilities
```

Comment:
I'd suggest to add a section for the tools themselves as a grouping:

- [Utilities](./Utilities)
  - [ConvertTo-ARMTemplate](./UtilitiesConvertToARMTemplate)  

Also, we should have the `Utilities` parent chapter in the sidebar. Not the child though as it may get a bit crowded over time

Reply:
done